classmodel/class-web
# Integration tests

Fixes #50 

TODO
* [x] automate scenario steps of issue in a test
* [x]  fix code to pass test
* [x] test other permutation actions
* [x] add aria attributes to make playwright tests more readable (and improve accessibility)


To test
```
cd app/class-solid
pnpm exec playwright test
pnpm exec playwright show-report
```
Expand `Duplicate experiment with a permutation` test and scroll to bottom to see the plot.

![d56cc5cc691adb07f801de3558bdad440fa833bd](https://github.com/user-attachments/assets/d716230d-5641-44ea-a7cd-72f33d698305)
